src/rules/terIndentRule.ts

```diff
@@ -1085,7 +1085,7 @@ class IndentWalker extends Lint.RuleWalker {
     } else {
       const nodeIndent = this.getNodeIndent(node).goodChar;
       const varKind = node.getFirstToken().getText();
-      const elementsIndent = nodeIndent + indentSize * OPTIONS.VariableDeclarator[varKind];
+      const elementsIndent = nodeIndent + indentSize * (OPTIONS.VariableDeclarator[varKind] || DEFAULT_VARIABLE_INDENT);
```

Comment:
Makes sense only in the case when `(OPTIONS.VariableDeclarator[varKind]` is undefined. But what about when we explicitly say that it should be zero? I think a better solution would be to modify the if statement in line 238:

```
OPTIONS.VariableDeclarator = {
          var: userOptions.VariableDeclarator,
          let: userOptions.VariableDeclarator,
          const: userOptions.VariableDeclarator
        };
```

We need to make sure that if `userOptions.VariableDeclarator` not a number then to set it to the default. Can you check if this would still work with your test?

Reply:
@jmlopez-rod `userOptions.VariableDeclarator` has default value yet.
I found the `varKind` is `export` (in my test), so `OPTIONS.VariableDeclarator[varKind]` is undefined.
Maybe best way is to find the correct varKind, but I'm not good at AST parser.
And you are right I should consider userOptions.VariableDeclarator is zero